src/bandersnatch/mirror.py

```diff
@@ -78,6 +78,13 @@ async def synchronize(
                 for name in specific_packages
             }
 
+        if not self.filters.filter_metadata_plugins:
+            logger.info("No metadata filters are enabled. Skipping metadata filtering")
+        if not self.filters.filter_release_plugins:
+            logger.info("No release filters are enabled. Skipping release filtering")
+        if not self.filters.filter_release_file_plugins:
+            logger.info("No release file filters are enabled. Skipping release file filtering")
+
```

Comment:
There should be only two semantic differences:

1. The origin of the message that is logged will be different, it'll come from mirror instead of package.  I don't see that as making a difference.
2.  Before, these 3 were sharing the same global variable, so I think once one message was logged, the others would *not* be logged, even though the messages are different and mean different things.  That feels erroneous to me

Reply:
Agree - As long as we're not not spamming the message for each package and we only say it once.